src/test/codewhisperer/commands/onInlineAcceptance.test.ts

```diff
@@ -0,0 +1,188 @@
+        })
+
+        it('Should format code selection with command vscode.executeFormatRangeProvider when current active document is not python', async function () {
+            const mockEditor = createMockTextEditor("console.log('Hello')", 'test.js', 'javascript', 1, 0)
+            const commandStub = sinon.stub(vscode.commands, 'executeCommand')
+            const extensionContext = await FakeExtensionContext.create()
+            const fakeReferences = [
+                {
+                    message: '',
+                    licenseName: 'MIT',
+                    repository: 'http://github.com/fake',
+                    recommendationContentSpan: {
+                        start: 0,
+                        end: 10,
+                    },
+                },
+            ]
+            await onInlineAcceptance(
+                {
+                    editor: mockEditor,
+                    range: new vscode.Range(new vscode.Position(0, 0), new vscode.Position(0, 27)),
+                    acceptIndex: 0,
+                    recommendation: "console.log('Hello World!')",
+                    requestId: '',
+                    sessionId: '',
+                    triggerType: 'OnDemand',
+                    completionType: 'Line',
+                    language: 'javascript',
+                    references: fakeReferences,
+                },
+                extensionContext.globalState
+            )
+            assert.ok(commandStub.calledWith('vscode.executeFormatRangeProvider'))
+        })
+
+        it('Should enqueue an event object to tracker', async function () {
+            const mockEditor = createMockTextEditor()
+            const trackerSpy = sinon.spy(CodeWhispererTracker.prototype, 'enqueue')
+            const extensionContext = await FakeExtensionContext.create()
+            const fakeReferences = [
+                {
+                    message: '',
+                    licenseName: 'MIT',
+                    repository: 'http://github.com/fake',
+                    recommendationContentSpan: {
+                        start: 0,
+                        end: 10,
+                    },
+                },
+            ]
+            await onInlineAcceptance(
+                {
+                    editor: mockEditor,
+                    range: new vscode.Range(new vscode.Position(1, 0), new vscode.Position(1, 26)),
+                    acceptIndex: 0,
+                    recommendation: "print('Hello World!')",
+                    requestId: '',
+                    sessionId: '',
+                    triggerType: 'OnDemand',
+                    completionType: 'Line',
+                    language: 'python',
+                    references: fakeReferences,
+                },
+                extensionContext.globalState
+            )
+            const actualArg = trackerSpy.getCall(0).args[0]
+            assert.ok(trackerSpy.calledOnce)
```

Comment:
Would it be possible to use `CodeWhispererTracker.flush()` + `assertTelemetry` instead of a spy?

Reply:
This test case checks whether an object is enqueued into the tracker, however, what is does is almost duplicating with test case  `it('Should report telemetry that records this user decision event'` . So I removed this enqueue check test case.

Test case `it('Should report telemetry that records this user decision event'` already covers `assertTelemetry`.